packages/generator/package.json

```diff
@@ -50,6 +50,7 @@
     "fast-xml-parser": "^4.0.11",
     "fs-extra": "^10.0.0",
     "glob": "^8.0.1",
+    "prettier": "^2.7.1",
```

Comment:
[req] This seems unnecessary? I didn't find any use in the generator package, only generator-common.

Reply:
ups, yes